src/core/dictionary.cpp

```diff
@@ -24,13 +25,75 @@ void Dictionary::remove(const std::string& key) { data_.erase(key); }
 
 bool Dictionary::found(const std::string& key) const { return data_.find(key) != data_.end(); }
 
-std::any& Dictionary::operator[](const std::string& key) { return data_.at(key); }
+std::any& Dictionary::operator[](const std::string& key)
+{
+    try
+    {
+        return data_.at(key);
```

Comment:
Here it should call `operator[]()`

```suggestion
        return data_.operator[](key);
```

Reply:
Should the dictionary be able to add new elements?